docs/setup.md

```diff
@@ -139,14 +139,15 @@ brew install pinentry-mac
 echo "pinentry-program /usr/local/bin/pinentry-mac" >> ~/.gnupg/gpg-agent.conf
 ```
 
-### Ubuntu & Debian
+### Ubuntu, Debian, Deepin, Devuan, Kali Linux, Pardus, Parrot, Raspbian
 
-**WARNING**: The official Debian repositories contain a package named `gopass` that
-is not related to this project in any way. It's a similar tool with a completely
-independent implementation and feature set. We are aware of this issue but can not
-do anything about it.
+**WARNING**: The official Debian repositories (and dervied distributions) contain
+a package named `gopass` that is not related to this project in any way.
+It's a similar tool with a completely independent implementation and feature set.
+We are aware of this issue but can not do anything about it.
 
-When installing on Ubuntu or Debian you can either download the `deb` package and install manually or use our repository.
+When installing on Ubuntu or Debian you can either download the `deb` package
+and install manually or build from source.
```

Comment:
I'd say the recommended way should be to `go get` gopass, no ?

Reply:
Yes, that's what I mean by "or build from source". How would you change that sentence?

Reply:
Make it an anchor link maybe?
`[install manually or build from source](#installing-from-source)`

Reply:
sounds good